cli/src/pixl_cli/_database.py

```diff
@@ -114,11 +114,15 @@ def _add_new_image_to_session(extract: Extract, message: Message, session: Sessi
     return new_image
 
 
-def images_for_project(project_slug: str) -> list[Image]:
+def processed_images_for_project(project_slug: str) -> list[Image]:
     """Given a project, get all images in the DB for that project."""
     PixlSession = sessionmaker(engine)
     with PixlSession() as session:
         return cast(
             list[Image],
-            session.query(Image).join(Extract).filter(Extract.slug == project_slug).all(),
+            session.query(Image)
+            .join(Extract)
+            .filter(Extract.slug == project_slug)
+            .filter(not_(Image.pseudo_study_uid.is_(None)))
```

Comment:
ooh avoiding the `== None` is fun. Might be worth making a separate PR where you find any time we've done that and replace with nicer syntax

Reply:
Also think that maybe this should be where `exported_at` isn't empty instead of `pseudo_study_uid`. Feels like you want to know when something has been exported and that you should expect to find it

Reply:
oof good catch